docs/quickstarts/excel-quickstart-jquery.md

```diff
@@ -8,8 +8,71 @@ ms.localizationpriority: high
 
 # Build an Excel task pane add-in
 
-In this article, you'll walk through the process of building an Excel task pane add-in with Yo Office.
+In this article, you'll walk through the process of building an Excel task pane add-in.
 
+You can create an Office Add-in by using the Office Add-ins Development Kit or the Yeoman generator in Visual Studio Code. Select the tab for the one you'd like to use and then follow the instructions to create your add-in and test it locally.
+
+# [Office Add-ins Development Kit](#tab/devkit)
+## Prerequisites
```

Comment:
The prerequisite section is same as the one section in (docs/develop/development-kit-overview.md)